Trevorcummings ccs create data

trevorcummingsCCS opened the pull request:
PR for the files used to insert into Agreement Service.

trevorcummingsCCS committed:
Added Tech Products

Saving just to be safe

trevorcummingsCCS committed:
Added additional 10 Commercial Agreements

trevorcummingsCCS committed:
added file name to header

trevorcummingsCCS committed:
Adding the sector data

adrianmilne reviewed (approved):
👍 Loaded into Agreements database with no issues

trevorcummingsCCS committed:
This file holds the route to market data

trevorcummingsCCS committed:
Just setting files up to edit later

trevorcummingsCCS committed:
Just sample data to base the file on

trevorcummingsCCS committed:
Initial set up.

Initial set up and added lots for RM3733 and RM6068

adrianmilne commented on agreements/create_lots.sql:
```diff
@@ -0,0 +1,201 @@
+ start_date,
+ end_date)
+values 
+(4,
```
Duplicate key - id 4 already exists (I will change to `10` locally to continue testing)

adrianmilne commented on agreements/create_lots.sql:
```diff
@@ -0,0 +1,201 @@
+ 'Technology Hardware',
+ 'Technology Hardware description',
+ 1,
+ 'Products',
```
According to Dave's API spec - Lot type is singular - i.e. 'product', 'service' or 'product and service'. Having them the same make mapping easier if possible 
![image](https://user-images.githubusercontent.com/2317320/84717584-262d7c80-af6e-11ea-9659-930f739ab459.png)

trevorcummingsCCS committed:
Committing the changes for RM6103

trevorcummingsCCS committed:
RM3808 had the wrong lot number

trevorcummingsCCS committed:
Commit changes for rm3808

trevorcummingsCCS committed:
Commit RM6154 lots

trevorcummingsCCS committed:
Committing RM3788

trevorcummingsCCS committed:
Committing RM3786

trevorcummingsCCS committed:
All Lots created

This is ready to be tested

trevorcummingsCCS committed:
Updated Ed to ED

trevorcummingsCCS committed:
All Lot sectors created

Will probably have to remove some but this will get people testing

trevorcummingsCCS committed:
commit rm3733 that is 6 lots out of 65

trevorcummingsCCS committed:
Added additional route to market

- aggregation

trevorcummingsCCS committed:
Commit changes before doing a global find and replace

trevorcummingsCCS committed:
Updated values

Changed default values I was using for 

lot_minimum_value,
lot_maximum_value, 
lot_contract_length_uom, 
lot_contract_length_minimum_value,
lot_contract_length_maximum_value

from 1,10000,'Month',6,12 to null,null,null,null,null

adrianmilne commented on agreements/create_lot_route_to_market.sql:
```diff
@@ -0,0 +1,855 @@
+  lot_minimum_value,lot_maximum_value, lot_contract_length_uom, lot_contract_length_minimum_value,lot_contract_length_maximum_value)
+ VALUES
+ (23,'Aggregation','12-Aug-2019','16-Aug-2022','National', 'https://www.crowncommercial.gov.uk/agreements',
+  null,null,null,null,null; 
```
Missing closing bracket `)`

adrianmilne commented on agreements/create_lot_route_to_market.sql:
```diff
@@ -0,0 +1,855 @@
+  lot_minimum_value,lot_maximum_value, lot_contract_length_uom, lot_contract_length_minimum_value,lot_contract_length_maximum_value)
+ VALUES
+ (51,'Further Competition','01-Jun-2017','01-Jun-2021','National', 'https://www.crowncommercial.gov.uk/agreements',
+  null,null,null,null,null;  
```
Missing closing bracket again `)`

adrianmilne commented on agreements/create_lot_sectors.sql:
```diff
@@ -0,0 +1,553 @@
+https://github.com/Crown-Commercial-Service/ccs-scale-db-scripts/*
```
Missing opening comment `/*`

trevorcummingsCCS committed:
Fixed syntax errors

trevorcummingsCCS committed:
Removed erroneous text from top of the file.
